src/app/app.routes.ts

```diff
@@ -0,0 +1,47 @@
+import { provideRouter, RouterConfig } from '@angular/router';
+import { MapViewComponent } from './views/map-view/map-view.component';
+import { CatViewComponent } from './views/cat-view/cat-view.component';
+import { CatHelpComponent } from './widgets/cat-help/cat-help.component';
+import { CatSource2FHLComponent } from './widgets/cat-source/cat-source-2fhl/cat-source-2fhl.component';
+import { CatSource3FGLComponent } from './widgets/cat-source/cat-source-3fgl/cat-source-3fgl.component';
+import { CatSourceSNRcatComponent } from './widgets/cat-source/cat-source-snrcat/cat-source-snrcat.component';
```

Comment:
@cdeil - check out `system-config.ts`, this is where angular-cli generates all barrels. If we want to make our own barrels, I'll have to add them in myself in this file. However, there's a scary "Everything underneath this line is managed by the CLI" message so I'm not sure if I should manually configure barrels.

For this PR, I'm not going to do anything about the barrels and I'll just leave all of the generated `index.ts` files.